_content/references.md

```diff
@@ -1,5 +1,155 @@
+| Agora On-premise Recording SDK               | On-premise Recording SDK   | Record and save voice calls, video calls, and interactive broadcasts to a server. |
+| Agora RTC SDK                                | RTC SDK                    | "Real-time Communications". We refer to all the Agora SDKs that enable real-time communication as the RTC SDK. |
+| Agora Real-time Messaging SDK, Agora RTM SDK | RTM SDK                    | Used to create a stable messaging mechanism for real-time messaging scenarios. Successor to the Agora Signaling SDK. |
+| Agora SDK                                    | Agora SDK                  | A generic term for referring to any Agora SDK. Preference is to specify the name of the SDK. |
+| Agora SD-RTN™                                | SD-RTN™                    | "Software-defined Real-time Network". Must appear with trademark symbol, preferred version of term |
+| Agora Video SDK                              | Video SDK                  | Used to embed video calling features into an app.            |
+| Agora RTC Web SDK                            | Web SDK                    | Uses APIs in the web browser to establish connections and control the communication and live broadcast services. |
+| Agora Interactive Whiteboard SDK             | Interactive Whiteboard SDK | Provides a simple collaboration platform on a virtual whiteboard where users from different locations can draw, annotate, and share PDF documents. |
+| Agora, Inc.                                  | Agora                      | Official company name.                                       |
+| Agora.io                                     | Agora                      | Variant of company name. Not used in technical documentation, with the exception of copyright statements (for example, "© 2016 Agora.io. All rights reserved.") |
+
+#### A
+
+- **add-on** (noun, adjective), **add on** (verb)
+- **Android device**
+- **app** (lowercase except when beginning a sentence or used as a proper noun)
+- **audio sample**: Use instead of *audio capture*.
+- **audio sample rate**
+
+#### B
+
+- **backend**: Use instead of *back-end* or *back end*.
+- **beta**
+- **bitrate**
+- **Boolean** (uppercase when referring to the datatype)
+- **Bluetooth**
+- **bridging header file**
+
+#### C
+
+- **case-sensitive**
+
+#### D
+
+- **data packet**
+- **dual-stream mode**
+- **dual stream** (when not modifying a noun)
+
+#### E
+
+- **email**
+- **earlier**: Use instead of *lower* when describing a version.
+- **extract**: Use instead of *unarchive* or *uncompress*.
+
+#### F
+
+- **filename**
+- **fill in; fill out**: Use *fill in* when referring to entering information in individual fields. Use *fill out* when referring to completing an entire form.
+- **front camera**
+
+#### G
+
+- **get**: Use instead of *fetch* or *retrieve*.
```

Comment:
For non-native English speakers, retrieve might be an unfamiliar word.

Reply:
According to Google's style guide, using technical keywords as English verbs is discouraged, see [Keywords](https://developers.google.com/style/code-in-text#keywords). So the verb "get" will not be confused with the method "GET". But I can see the need for different terms to avoid repetition in some cases. Maybe we can just remove this.